### ISSUE TITLE
create conda-forge recipe

### ISSUE THREAD

**Issue by quasiben** [dask/old-dask-yarn#16]
cc @martindurant

**Comment by martindurant** [comment]
I notice that there is already a conda recipe - does this build correctly? In that case, I would imagine it being very simple to make a feedstock (replacing source='..' with a git checkout).

**Comment by quasiben** [comment]
yup yup, the conda recipe is correct and it's what I use when uploading packages to anaconda.org
